.github/workflows/ci.yml

```diff
@@ -12,6 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - run: |
+          rustup override set 1.76.0
+          rustup component add rustfmt
+          rustup component add clippy
+          rustup component add rust-docs
```

Comment:
why do we need the override?

Reply:
Because the action by itself installs the latest version of rust